deploy/build-dex.Jenkinsfile

```diff
@@ -0,0 +1,71 @@
+def legion
+def commit_id = null
+def dockerCacheArg
+
+pipeline {
+    agent any
+
+    options{
+        buildDiscarder(logRotator(numToKeepStr: '5'))
+        disableConcurrentBuilds()
+    }
+    parameters {
+        booleanParam(defaultValue: false, description: 'Enable slack notifications', name: 'EnableSlackNotifications')
+        booleanParam(defaultValue: false, description: '', name: 'EnableDockerCache')
+    }
+    environment {
```

Comment:
Currently all such properties are stored in job parameters section and not commited to git within pipeline.
So we don't share our endpoint or resources names or anything else. 
Shouldn't we follow some common way and set this properties in jenkins job as parameters the same way?
@kirillmakhonin @dsuslov

Reply:
Has Removed parameters in pipeline,  configured them in job parameters section.